examples/v1alpha1/pipelineruns/conditional-pipelinerun-continue-after-skip-ordering.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Condition
+metadata:
+  name: always-false
+spec:
+  check:
+    image: alpine
+    script: |
+      exit 1 
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: echo-expected
+spec:
+  steps:
+    - name: echo-expected
+      image: ubuntu
+      script: 'echo expected'
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: echo-unexpected
+spec:
+  steps:
+    - name: echo-file-exists
+      image: ubuntu
+      script: 'echo UNEXPECTED!'
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: conditional-pipeline
+spec:
+  tasks:
+    - name: task-should-be-skipped # failed
+      conditions:
+        - conditionRef: always-false
+      continueAfterSkip: 'true' # allows executing dependent tasks
+      taskRef:
+        name: echo-unexpected
+    - name: task-should-execute # succeeded
+      taskRef:
+        name: echo-expected
+      runAfter: 
+        - task-should-be-skipped
```

Comment:
my understanding is that this pipeline run will succeed even if continueAfterSkip doesn't work properly - maybe we can add a finally task that verifies that task-should-execute executed? e.g. (until we have #2557!) we could write to a workspace in this task, and the finally task could verify that the workspace was written to?